lib/bamboo/phoenix.ex

```diff
@@ -114,6 +112,15 @@ defmodule Bamboo.Phoenix do
   end
 
   @doc """
+  Sets the layout for rendering plain text and HTML templates
```

Comment:
While we have this open, could you add examples to the layout functions? Something simple like 

```elixir
# Will use MyApp.LayoutView with email.html and email.text layouts when rendering
put_layout(email, {MyApp.LayoutView, :email})
```

What do you think?